vip-helpers/vip-utils.php

```diff
@@ -1494,24 +1500,10 @@ function vip_is_jetpack_request() {
 	require_once __DIR__ . '/../lib/proxy/class-iputils.php';
 
 	// If has a valid-looking UA, check the remote IP
-	// From https://jetpack.com/support/hosting-faq/#jetpack-whitelist
-	// Or https://jetpack.com/ips-v4.json
-	$jetpack_ips = array(
-		'122.248.245.244/32',
-		'54.217.201.243/32',
-		'54.232.116.4/32',
-		'192.0.80.0/20',
-		'192.0.96.0/20',
-		'192.0.112.0/20',
-		'195.234.108.0/22',
-		'192.0.96.202/32',
-		'192.0.98.138/32',
-		'192.0.102.71/32',
-		'192.0.102.95/32',
```

Comment:
These ones belong to the 192.0.96.0/20 range.

I assume it is safe to remove them?